src/Teams/Services/ManageSprintsService.cs

```diff
@@ -22,7 +23,7 @@ public ManageSprintsService(IRepository<Sprint, int> sprintRepository, IManageTe
 
         public async Task<IEnumerable<Sprint>> GetAllSprintsAsync(int team_id, DisplayOptions options)
         {
-            var sprints = _sprintRepository.GetAll().Include(x => x.Team).Where(x => x.TeamId == team_id);
+            var sprints = _sprintRepository.GetAll().Include(x => x.Team).Include(x=>x.Tasks).Where(x => x.TeamId == team_id);
```

Comment:
Do you always need tasts included?
Maybe better to propose here bool option to include it or not?